README.md

```diff
@@ -60,22 +60,46 @@ A third-party plugin is required to enable syntax coloring. The Che4z basic stac
 	
 ## Copybook Support
 
-The LSP for COBOL extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can open copybooks in your IDE and make use of the copybook support features of the extension.
+The COBOL Language Support extension supports copybooks called by your source code that are stored in a local folder in your workspace. If your copybooks are stored in mainframe data sets, you can use a Zowe CLI z/OSMF profile to automatically download them from the mainframe to your workspace. 
 
-#### Retrieving Copybooks
+To enable copybook support, you specify the local folders and mainframe data sets that contain copybooks used in your project in the workspace settings. When a copybook is called in the program, the folders and data sets are searched in the order they are listed for files and members that match the copybook's name. If a copybook with the same file name is located in both a local folder and a mainframe data set, the one in the local folder is used.
```

Comment:
It is technically correct, but a little misleading. Retrieving the copybooks from a mainframe will work even without specifying a local folder - they will appear in the predefined one.

Reply:
Thanks, I've just tested it and this is exactly what happens. Fixed in latest commit.

Reply:
Any clearer now?

Reply:
Maybe, "...you *may* specify the local folders..." will be enough to show that it is not obligatory. The following text explains the process well.

Reply:
It is obligatory to specify either folders or data sets. I hope the new opening sentence makes that clear enough. Adding "can" or "may" just makes it sound like the whole thing is optional.

Reply:
Ok, I agree